modules/certificate/certificates.permission.js

```diff
@@ -20,7 +20,7 @@ const {
 
 const certificatePermissionsQuery = {
   getCertificateById: hasRoles([ADMIN, SUPERADMIN]),
-  getCertificateByName: allow,
```

Comment:
I guess "allow" is the default behavior, you can remove useless line

Reply:
fixed